monGARS/core/dynamic_response.py

```
@@ -107,46 +130,37 @@ async def get_personality_traits(
+                user_id,
+            )
+
+        try:
+            adapted = self._style_tuner.apply_style(
+                user_id,
+                text,
+                dict(personality or {}),
+            )
+        except Exception as exc:  # pragma: no cover - defensive fallback
+            logger.exception(
+                "Failed to apply style adaptation for %s: %s", user_id, exc
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)